{
  "_annotation": {
    "cve_id": "CVE-2022-21278",
    "generated_from": "https://raw.githubusercontent.com/anchore/cve-data-enrichment/main/data/anchore/2022/CVE-2022-21278.json",
    "reason": "Improve version ranges to indicate fix"
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:oracle:mysql:*:*:*:*:*:*:*:*",
                "matchCriteriaId": "EFD22AB1-E391-5814-8305-322ABCF0E76D",
                "versionEndExcluding": "8.0.27",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          },
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:oracle:mysql_server:*:*:*:*:*:*:*:*",
                "matchCriteriaId": "F98E974D-ECEC-5BD5-96AF-DA2615C1EA78",
                "versionEndExcluding": "8.0.27",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}